lib/puppet/functions/postgresql/postgresql_password.rb

```diff
@@ -6,15 +6,25 @@
   #   The clear text `username`
   # @param password
   #   The clear text `password`
+  # @param sensitive
+  #   If the Postgresql-Passwordhash should be of Datatype Sensitive[String]
   #
-  # @return [String]
+  # @return
   #   The postgresql password hash from the clear text username / password.
   dispatch :default_impl do
-    param 'Variant[String[1],Integer]', :username
-    param 'Variant[String[1],Integer]', :password
+    required_param 'Variant[String[1], Integer]', :username
+    required_param 'Variant[String[1], Sensitive[String[1]], Integer]', :password
```

Comment:
You are removing the `Integer` as an input value, which was valid before.
```suggestion
    required_param 'Variant[String[1], Integer, Sensitive[String[1]], Integer]', :password
```